src/ast/async_event_types.h

```diff
@@ -59,5 +59,19 @@ struct Time
   }
 } __attribute__((packed));
 
+struct Buf
+{
+  uint8_t length;
+  uint8_t* content;
```

Comment:
Since this is how you're going to cast it next commit anyways
```suggestion
  char* content;
```

Reply:
Did you mean `char content[];`? Not sure how to square this with the suggestion about FLA and being less misleading with the type.

Reply:
Ah yeah, sorry. Should be `char content[]`